udata_gouvfr/theme/gouvfr/translations/fr/LC_MESSAGES/gouvfr@@.po

```diff
@@ -0,0 +1,346 @@
+msgstr "Se connecter"
+
+#: templates/header.html:53
+msgid "Sign in"
+msgstr "Se connecter"
+
+#: templates/header.html:56
+msgid "Register"
+msgstr "S'enregistrer"
+
+#: templates/header.html:56
+msgid "Sign up"
+msgstr "S'enregistrer"
+
+#: templates/home.html:18
+msgid "Open platform for French public data"
+msgstr "Plateforme ouverte des données publiques Françaises"
+
+#: templates/home.html:20
+msgid "Featured topics"
+msgstr "Thématiques à la une"
+
+#: templates/home.html:22
+msgid "Données de"
+msgstr "Données de"
+
+#: templates/home.html:23
+msgid "Référence"
+msgstr "Référence"
+
+#: templates/home.html:27
+msgid "Données relatives à la"
+msgstr "Données relatives à la"
+
+#: templates/home.html:28
+msgid "COVID-19"
+msgstr "COVID-19"
+
+#: templates/home.html:32
+msgid "Données relatives aux"
+msgstr "Données relatives aux"
+
+#: templates/home.html:33
+msgid "Comptes Publics"
+msgstr "Comptes Publics"
+
+#: templates/home.html:38
+msgid "News"
+msgstr "Actualités"
+
+#: templates/home.html:46
+msgid "All news"
+msgstr "Toutes les actualités"
+
+#: templates/home.html:68
+msgid "is"
+msgstr "c’est"
+
+#: templates/home.html:87
+msgid "Reference datasets"
+msgstr "Jeux de données de référence"
+
+#: templates/home.html:95
+msgid "Data"
+msgstr "Données"
+
+#: templates/home.html:127
+msgid "Etalab selection"
+msgstr "Selection Etalab"
+
+#: templates/dataset/card.html:32 templates/dataset/search-result.html:69
+msgid "resources"
+msgstr "ressources"
+
+#: templates/dataset/card.html:33 templates/dataset/search-result.html:71
+#: templates/organization/card.html:27
+msgid "reuses"
+msgstr "réutilisations"
+
+#: templates/dataset/display.html:106 templates/dataset/display.html:203
+msgid "Extras"
+msgstr "Extras"
+
+#: templates/dataset/explore.html:37
+msgid "Open platform for French public data"
+msgstr "Plateforme ouverte des données publiques françaises"
+
+#: templates/dataset/list.html:32 templates/reuse/list.html:41
+msgid "Search..."
+msgstr "Rechercher..."
+
+#: templates/dataset/list.html:36 templates/organization/display.html:55
+#: templates/organization/display.html:87
+msgid "Datasets"
+msgstr "Jeux de données"
+
+#: templates/dataset/list.html:38
+msgid "Search reuses"
+msgstr "Rechercher dans les réutilisations"
+
+#: templates/dataset/search-result.html:74
+msgid "followers"
+msgstr "abonnés"
+
+#: templates/dataset/resource/card.html:10
+msgid "Edit button"
+msgstr "Bouton d'édition"
+
+#: templates/dataset/resource/card.html:16
+msgid "downloads"
+msgstr "téléchargements"
+
+#: templates/dataset/resource/card.html:28
+msgid "See more details"
+msgstr "Voir plus de détails"
+
+#: templates/dataset/resource/card.html:40
+msgid "Resource link"
+msgstr "Lien de la ressource"
+
+#: templates/dataset/resource/card.html:42
+msgid "Download resource"
+msgstr "Télécharger la ressource"
+
+#: templates/dataset/resource/card.html:65
+msgid "URL"
+msgstr "URL"
+
+#: templates/dataset/resource/card.html:69
+msgid "Stable URL"
```

Comment:
```suggestion
msgid "Permalink"
```

Reply:
Not sure, we're using "URL stable" a lot when talking to producers, and at least it's "real" french.

Reply:
Ok :+1: